core-server/pom.xml

```diff
@@ -169,6 +169,7 @@
         <dependency>
             <groupId>javax.ws.rs</groupId>
             <artifactId>javax.ws.rs-api</artifactId>
+            <version>2.2-SNAPSHOT</version>
```

Comment:
This should be in pom.xml rather than in core-server/pom.xml?

Reply:
Right! In fact it already exists in pom.xml so it simply is useless in core-server/pom.xml.